pkg/importer/util.go

```diff
@@ -38,10 +39,12 @@ func MoveFile(src, dst string) error {
 	return err
 }
 
-// StreamDataToFile provides a function to stream the specified io.Reader to the specified local file
+// StreamDataToFile provides a function to stream the specified io.Reader to the specified local file or block device
```

Comment:
Since you are no longer using this to write to a block device, the comment shouldn't indicate it is.

Reply:
done